pkg/cluster/managed.go

```diff
@@ -76,6 +78,10 @@ func (c *Cluster) start(ctx context.Context) error {
 		return fmt.Errorf("cluster-reset was successfully performed, please remove the cluster-reset flag and start %s normally, if you need to perform another cluster reset, you must first manually delete the %s file", version.Program, resetFile)
 	}
 
+	if _, err := os.Stat(resetFile); err == nil {
+		// before removing reset file we ned to wipe the node passwd secret
+		go c.deleteNodePasswdSecret(ctx)
```

Comment:
Does this need to be in a goroutine?

Reply:
yes it needs to since the api will not be started yet